packages/devtools_app/lib/src/screens/debugger/codeview_controller.dart

```diff
@@ -42,8 +48,19 @@ class CodeViewController extends DisposableController
 
   @override
   void dispose() {
-    super.dispose();
     scriptsHistory.current.removeListener(_scriptHistoryListener);
+    _scriptLocation.dispose();
```

Comment:
Assuming these all implement `Disposable`, can we add a `List<Disposable> get disposables = [];` to the `DisposableController` that can be overridden with a list or a set of properties that need to be disposed so we can just loop over them in the `DisposableController`'s `dispose()`?

Reply:
Unfortunately, there is not a common `Disposable` base class that would allow us to do that.